queries/clojure.scm

```diff
@@ -1,3 +1,11 @@
 (comment) @comment @textFragment
 
 (str_lit) @string @textFragment
+
+(map_lit) @map
+
+;; A list is either a vector literal or a quoted list literal
+[
+  (vec_lit)
+  (quoting_lit)
```

Comment:
you seem to have dropped the `.list_lit` child here?

Reply:
fixed